frontend/src/components/MissionOverview/UpcomingMissionCard.tsx

```diff
@@ -113,7 +114,13 @@ function MissionStartTimeDisplay({ mission }: MissionDisplayProps) {
     return (
         <StyledMissionStartTime>
             <Typography>Date: {format(new Date(mission.startTime), 'dd. MMM')}</Typography>
-            <Typography>Time: {format(new Date(mission.startTime), 'hh:mm')}</Typography>
+            <Typography>
+                Time:{' '}
+                {new Date(mission.startTime).toLocaleTimeString('no-NO', {
```

Comment:
This could also use the "default time", but chose to use Norwegian

Reply:
Good choice